ui/src/wizard/ResourceGroupListStep.ts

```diff
@@ -79,7 +79,7 @@ export class ResourceGroupListStep<T extends types.IResourceGroupWizardContext>
         }
 
         const resourceGroups: ResourceGroup[] = await ResourceGroupListStep.getResourceGroups(wizardContext);
-        return picks.concat(resourceGroups.map((rg: ResourceGroup) => {
+        picks = picks.concat(resourceGroups.map((rg: ResourceGroup) => {
```

Comment:
I would sort the resource groups here before you turn them into picks, instead of sorting the picks at the end. This will ensure "Create new resource group" is always at the top